registry/serverless.yml

```diff
@@ -0,0 +1,243 @@
+plugins:
+  - serverless-offline
+  - serverless-python-requirements
+  - serverless-plugin-tracing
+  - serverless-prune-plugin
+
+service: registry
+provider:
+  name: aws
+  runtime: python3.6
+  description: offchain information of organizations and services
+  memorySize: 512
+  timeout: 30
+  region: ${file(./config.${self:provider.stage}.json):REGION}
+  stage: ${opt:stage,'dev'}
+  deploymentBucket:
+    name: snet-serverless-artifacts
+    serverSideEncryption: AES256
+    tags:
+      key1: registry
+    deploymentPrefix: serverless
+    tracing: true
+
+custom:
+  pythonRequirements:
+    fileName: contract_api/requirement.txt
+    dockerizePip: true
+    useDownloadCache: true
+    useStaticCache: true
+    cacheLocation: "/var/cache/serverless"
+  prune:
+    automatic: true
+    includeLayers: true
+    number: 5
+
+package:
+  exclude:
+    - .circleci/**
+    - .gitignore/**
+    - .serverless/**
+    - requirement.txt
+    - venv/**
+    - config.ropsten.json
+    - sls_deploy.sh
+    - serverless.yml
+    - test/**
+    - tests/**
+    - sql_script/**
+    - service_status/**
+    - dapp-user/**
+    - repository/**
+    - Readme.md
+    - parse_events.sh
+    - package.json
+    - Dockerfile
+    - License
+    - log_setup.py
+    - heath_check.sh
+    - node_modules/**
+    - wallets/**
```

Comment:
lets exclude everythin and includer what we need , that we will not need to chenage exclude list if some one adds somehting